bin/external_lib_deps.ml

```diff
@@ -131,10 +131,11 @@ let term =
     Arg.(value & flag & info [ "sexp" ] ~doc:{|Produce a s-expression output|})
   in
   Common.set_common common ~targets:[];
+  Dune.Clflags.external_lib_deps_mode := true;
```

Comment:
Why is this set differently, i.e. not as `Common.set_common common ~targets:[] ~external_lib_deps_mode:false;`? If this is a special case, perhaps, it deserves a comment.

Reply:
Good catch. Fixed now.